### DISCUSSION TITLE
Possible bandaid for missing font Gentium Plus on Mac

### DISCUSSION THREAD

**Discussion by ethanwood17** [sile-typesetter/sile#1603] [Ideas]
I recently installed Sile for the first time and tried to format the Hello World example shown in the Sile book. I was immediately greeted with an error about missing the font Gentium Plus. I saw the discussion about it on #369 and looked for a way to fix it.

I looked into making the font Gentium Plus a dependency in the Homebrew formula. Unfortunately, for reasons I don't fully understand, the Homebrew support for adding a cask as a dependency seems spotty. Adding font-gentium-plus doesn't work, at least in the ways I tried. 

An alternative that might give users a heads up is to add a [Caveat](https://docs.brew.sh/Formula-Cookbook#caveats). I added a caveat to my local sile.rb file that calls out the need to install Gentium Plus. Here's what that looks like: 

```
==> Caveats
Sile uses the font Gentium Plus by default. If this font is not installed or an alternative is not specified, you may encounter errors.

To install Gentium Plus from a Homebrew Cask run:
  brew install --cask font-gentium-plus
```

Would something like that be an acceptable solution? I generally read and pay attention to Homebrew caveats, so in my case I think it would have helped me. If we ever change the default font or fix this issue, the caveat can be removed. I would also be happy to submit a PR for that, if the community/maintainers think it's a good idea.

**Comment by alerque** [comment]
Yes that sounds great. I didn't know about the caveat system in Homebrew.

**Comment by Omikhleia** [comment]
@ethanwood17 @alerque Should this be converted to an issue or was it addressed?

**Reply by alerque** [reply]
The Homebrew package has had a relevant Caveat for some time, this is addressed.
